xml/x509_subjects.xml

```diff
@@ -168,6 +168,14 @@
     <param pos="0" name="os.product" value="iDRAC Linux"/>
   </fingerprint>
 
+  <fingerprint pattern="^CN=opensnort\.example.com,O=Sourcefire,emailaddress=support@sourcefire.com">
+    <description>Sourcefire Snort HTTPS console</description>
+    <example>CN=opensnort.example.com,O=Sourcefire,emailaddress=support@sourcefire.com</example>
+    <param pos="0" name="service.vendor" value="Snort"/>
+    <param pos="1" name="service.family" value="Snort"/>
```

Comment:
should this be pos 1? We are not using a capture group

Reply:
Correct